operate/webapp/src/main/java/io/camunda/operate/webapp/backup/BackupRepository.java

```diff
@@ -23,4 +30,25 @@ public interface BackupRepository {
 
   void executeSnapshotting(
       BackupService.SnapshotRequest snapshotRequest, Runnable onSuccess, Runnable onFailure);
+
+  default boolean isIncompleteCheckTimedOut(
+      final OperateProperties operateProperties,
```

Comment:
Since we only use the property operateProperties.getBackup().getIncompleteCheckTimeoutInSeconds(), I would rather only pass this property as parameter, instead of the whole OperateProperties object.